lib/nepali_calendar/ad_calendar.rb

```diff
@@ -111,5 +111,9 @@ def date_range
         start_date.end_of_month.end_of_week
       ]
     end
+
+    def to_date
```

Comment:
this code is in https://github.com/Daanphe/nepali_calendar/pull/13 already

are we supposed to be repeating this here... if it is not necessary then let's isolate the codes

Or we can close the PR#13 and go on with this one